### PR TITLE
fix: k8s policies tutorial

### PR DESCRIPTION
**Pull request by josedonizetti**
### 1. Explain what the PR does

Fix https://github.com/aquasecurity/tracee/issues/3368

"Replace me with `make check-pr` output"

### 2. Explain how to test it

```
make -f builder/Makefile.mkdocs
```

### CODE REVIEW

File: docs/tutorials/k8s-policies.md [hunk 1]
```diff
@@ -106,23 +106,26 @@ Let's edit the tracee-policies ConfigMap using kubectl:
 kubectl edit cm/tracee-policies -n tracee-system
 ```
 
-The ConfigMap will open in your default text editor. Locate the signatures.yaml section.
+The ConfigMap will open in your default text editor. Locate the data section.
 
-To add a new policy for tracking execve events, append the following YAML block under the signatures.yaml section, maintaining proper indentation:
+To add a new policy for tracking execve events, add the following YAML block before the signaturesy.yaml section, maintaining proper indentation:
```
Comment (AnaisUrlichs) [inline docs/tutorials/k8s-policies.md:111]: signature spelling mistake

File: docs/tutorials/k8s-policies.md [hunk 2]
```diff
@@ -106,23 +106,26 @@ Let's edit the tracee-policies ConfigMap using kubectl:
...
+          description: traces all signature events
+    spec:
+        scope:
+          - global
+        rules:
+          - event: execve
+  signatures.yaml: |-
+  ...
 ```
 Save and close the file. The changes will be applied automatically.
```
Comment (AnaisUrlichs) [inline docs/tutorials/k8s-policies.md:130]: But below it says to restart the Daemonset? Is that still required?

**Review by AnaisUrlichs — COMMENTED** [review]
added a few comments, otherwise, looks good

File: docs/tutorials/k8s-policies.md (hunk 2 above)
Comment (josedonizetti) [reply docs/tutorials/k8s-policies.md:130]: yes restarting is required, maybe `automatically` pass the wrong idea here

File: docs/tutorials/k8s-policies.md [hunk 3]
```diff
@@ -106,23 +106,26 @@ Let's edit the tracee-policies ConfigMap using kubectl:
 kubectl edit cm/tracee-policies -n tracee-system
 ```
 
-The ConfigMap will open in your default text editor. Locate the signatures.yaml section.
+The ConfigMap will open in your default text editor. Locate the data section.
```
[the remark this answers is not in the document]
Comment (josedonizetti) [reply docs/tutorials/k8s-policies.md:109]: are you running the edit command? `kubectl edit cm/tracee-policies -n tracee-system`

**Comment by josedonizetti** [comment]
@AnaisUrlichs hey I have addressed your question, can take another look?

File: docs/tutorials/k8s-policies.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (AnaisUrlichs) [reply docs/tutorials/k8s-policies.md:109]: yes that makes sense hahah

**Comment by josedonizetti** [comment]
@AnaisUrlichs still waiting for your review to be able to merge this

**Review by AnaisUrlichs — APPROVED** [review]

File: docs/tutorials/k8s-policies.md [hunk 4]
```diff
@@ -106,25 +106,28 @@ Let's edit the tracee-policies ConfigMap using kubectl:
...
-	  rules:
-	    - event: execve
+data:
+  events.yaml: |-
+    apiVersion: aquasecurity.github.io/v1beta1
+    kind: TraceePolicy
+    metadata:
+        name: execve-event
+        annotations:
+          description: traces all signature events
```
Comment (yanivagman) [inline docs/tutorials/k8s-policies.md:121]: Should the description be related to the execution event?

Comment (josedonizetti) [reply docs/tutorials/k8s-policies.md]: fixed, can you take a nother look?

**Pull request merged**